setup.py

```diff
@@ -49,7 +49,7 @@ def _get_version_git():
 # Check version
 try:
     gver = _get_version_git()
-    if "-" in gver:
+    if b"-" in gver:
```

Comment:
Typically, in Python, the versions are "strings". So I'd suggest to just adjust _get_version_git() to return a string. => .decode("utf-8")  should do it.